### PR TITLE
Add attachment support

### PR DESCRIPTION
**Pull request by amanda11**
Add support to service now attachment actions of:

- GET attachments - returns the meta data of all attachments on the table

- GET attachment <sysid> - returns the meta data for single attachment (including the download link)

- POST attachment <sysid> - uploads an attachment to an entry in a table

### CODE REVIEW

File: actions/get_attachment.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+class GetAttachmentAction(BaseAction):
+    def run(self, table, sys_id):
+        s = self.client
+        api_path = "/attachment/{}".format(sys_id)
+        r = s.resource(api_path=api_path)
+        records = r.get(query={})
+        output = []
+        for record in records.all():
+            output.append(record)
+        return output
```
Comment (nmaludy) [inline actions/get_attachment.py]:
```suggestion
        return [r for r in records.all()]
```

File: actions/get_attachments.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from lib.actions import BaseAction
+
```
Comment (nmaludy) [inline actions/get_attachments.py:2]: This looks like the same code as what's above except no ID. Might be easier/cleaner to create a common function in the BaseAction and just call it from both action implementation files, that way there is less copy/paste going on?

File: pack.yaml [hunk 3]
```diff
@@ -5,7 +5,7 @@ description: ServiceNow Integration Pack
 keywords:
   - servicenow
   - incident management
-version: 0.3.8
```
Comment (nmaludy) [inline pack.yaml:8]: Should probably bump this to 0.4.0 since we're adding new features here.

**Review by nmaludy — CHANGES_REQUESTED** [review]
Couple small tweaks, looks good overall.

File: actions/get_attachments.py (hunk 2 above)
Comment (amanda11) [reply actions/get_attachments.py:2]: Great point - trouble when you develop them separately and then forget to work out what was common!

**Comment by amanda11** [comment]
@nmaludy Thanks for review comments, I think these have been addressed - and I also realised when addressing the refactor that table parameter on get was unused so removed that.

**Review by nmaludy — APPROVED** [review]

**Pull request merged**